class ForceError():
    def __init__(self):
        pass

    def force1():

        raise  RuntimeError('Mensagem aleatoria foda-se')
